packages/ruleset/test/no-superfluous-allof.test.js

```diff
@@ -0,0 +1,115 @@
+/**
+ * Copyright 2017 - 2024 IBM Corporation.
+ * SPDX-License-Identifier: Apache2.0
+ */
+
+const { noSuperfluousAllOf } = require('../src/rules');
+const { makeCopy, rootDocument, testRule, severityCodes } = require('./utils');
+
+const rule = noSuperfluousAllOf;
+const ruleId = 'ibm-no-superfluous-allof';
+const expectedSeverity = severityCodes.warning;
+const expectedMsg = 'Avoid schemas containing only a single-element allOf';
```

Comment:
Hmm I'm surprised that this message doesn't match the message provided in the function. Do you know why that message isn't being used?

Reply:
This was happening because the rule defn didn't specify the "message" field.  I apparently copy/pasted from the wrong existing rule (one that supplied the error message in the rule itself rather than the rule function) :joy: